utils/build_swift/tests/expected_options.py

```diff
@@ -69,7 +69,7 @@
     'build_playgroundsupport': False,
     'build_runtime_with_host_compiler': False,
     'build_stdlib_deployment_targets': ['all'],
-    'build_subdir': 'Ninja-DebugAssert',
```

Comment:
Why is this default gone?

Reply:
nevermind, I see that we're computing this if it's not passed.

Reply:
I've also moved the `build_subdir` inference logic back to `build-script` which means that the default value will be `None` and we'll infer it.